ExchangeWebServices/Docs/MS-OXWSCONT/[MS-OXWSCONT].md

```diff
@@ -1,6 +1,6 @@
 ## [MS-OXWSCONT]: Contacts Web Service Protocol
-- [MS-OXWSCONT] v20211116 is used to develop the test suite.
-- Release date, 11/16/2021.
+- [MS-OXWSCONT] v20220517is used to develop the test suite.
```

Comment:
Please leave a blank between the date and 'is'.

Reply:
Updated the blank between the date and 'is'